src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphEntityDAOSpec.scala

```diff
@@ -84,7 +84,7 @@ class GraphEntityDAOSpec extends FlatSpec with Matchers with OrientDbTestFixture
     assert {
       txn.withGraph { graph =>
         graph.getVertices("_clazz", "pair")
-          .filter(v => v.getProperty("_name") == "pair2")
+          .filter(v => v.getProperty("_name").asInstanceOf[Object] == "pair2")
```

Comment:
I think a better fix for this and the following is
v.getProperty[String]("_name") == "pair2")